releasenotes/notes/fix-kal-trace-agent-c0ed330fb13ebd87.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Improve connection reuse with http keep-alive in
+    trace agent.
```

We have a rule that we have to prefix changelog items for APM with `APM: `. Can you please do that?
```suggestion
    APM: Improve connection reuse with HTTP keep-alive in
    trace agent.
```